litellm/utils.py

```diff
@@ -930,6 +937,8 @@ def wrapper(*args, **kwargs):  # noqa: PLR0915
                     kwargs["retry_policy"] = (
                         reset_retry_policy()
                     )  # prevent infinite loops
+                    kwargs["max_retries"] = 0
+                    kwargs["num_retries"] = 0
```

Comment:
https://github.com/BerriAI/litellm/blob/1ef311343c93fdccec0a0e07a3079febcb9c433e/litellm/llms/OpenAI/openai.py#L546 appears to continually reset `max_retries` to `2` via the `pop()` logic every time the OpenAI provider is called, so we need to reset here after every iteration as well

Reply:
@dbczumar the max retries is set to this to handle the user/assistant alternating scenario for openai-compatible routes like Mistral (see the try except block)

Reply:
got it - so it looks like you want to disable the openai max retries param so we don't pass that to their client sdk